app/controllers/contacts_controller.rb

```diff
@@ -23,19 +23,18 @@ def new
   # GET /councils/1/edit
   def edit
   end
-
-  def mail
-    if (params[:name]) && (params[:email]) && (params[:msg])
+  def mail    
```

Comment:
Assignment Branch Condition size for mail is too high. [19.65/15]<br>Use empty lines between defs.<br>Unnecessary spacing detected.<br>Trailing whitespace detected.